server/src/infra/configuration.rs

```diff
@@ -131,6 +131,8 @@ pub struct Configuration {
     pub ldaps_options: LdapsOptions,
     #[builder(default = r#"HttpUrl(Url::parse("http://localhost").unwrap())"#)]
     pub http_url: HttpUrl,
+    #[builder(default = r#"String::from("./app")"#)]
+    pub assets_path: String,
```

Comment:
A few things here: 
- can you move it above the option groups? (Ldaps, SMTP, ...)
- can we parse it as a path instead of leaving it as a string? It's a bit too loose of a type for my taste. 
- maybe add a warning on startup if, let's say, index.html can't be found